rmc/utils/constraint.py

```diff
@@ -702,7 +718,9 @@ def search_for_break(
         f"{TEMP_PATH_WITH_FAST_DEL}/freeze{freeze}_round{search_num}_all_loci_chisq.ht"
     )
     if save_chisq_ht:
-        all_loci_chisq_ht_path = f"{SINGLE_BREAK_TEMP_PATH}/all_loci_chisq.ht"
+        all_loci_chisq_ht_path = (
+            f"{MODEL_PREFIX}/{CURRENT_GNOMAD_VERSION}/{freeze}/constraint_prep.ht"
+        )
```

Comment:
This is the path for `constraint_prep` resource - are you sure you want to write here?

Reply:
hm. this looks like a copy-paste (was trying to get the `constraint_prep` path and update the name) -- thanks!